src/frontends/tensorflow_common/src/op/matrix_inverse.cpp

```diff
@@ -0,0 +1,34 @@
+// Contributed by Hongbo Wei
+// https://github.com/hongbo-wei
+// Good first issue: https://github.com/openvinotoolkit/openvino/issues/22957
+
+#include "openvino/op/inverse.hpp"
+
+#include "common_op_table.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_matrix_inverse_op(const NodeContext& node) {
+    auto op_type = node.get_op_type();
+    TENSORFLOW_OP_VALIDATION(node, op_type == "MatrixInverse", "Internal error: incorrect usage of translate_matrix_inverse_op.");
+    auto equation = node.get_attribute<std::string>("equation"); 
+
+    OutputVector inputs;
+    for (size_t input_ind = 0; input_ind < node.get_input_size(); ++input_ind) {
+        inputs.push_back(node.get_input(input_ind));
+    }
```

Comment:
why do we need it? MatrixInverse always has one input

Reply:
Make sense.